### PR TITLE
Add new Japanese translations for kubectl

### PR DESCRIPTION
**Pull request by tsasaki609**
**What this PR does / why we need it**:
This PR add new Japanese translations for cubectl with merging from translations/kubectl/template.pot file.
Most of it is a copy from the source text, but a little translated.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #
No issues

**Special notes for your reviewer**:
I contribute for the first time.
Please let me know if there is something wrong.

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by tsasaki609** [comment]
I signed CLA.

**Comment by idealhack** [comment]
@tsasaki609 Thanks for your contribution! It seems that you did not sign the CLA, please read those comments :)

/ok-to-test

**Comment by tsasaki609** [comment]
I can't see the page signing CNCF CLA (https://identity.linuxfoundation.org/projects/cncf)  so I'm contacting the help desk.

**Comment by tsasaki609** [comment]
Now I know that adding a translation will generate go-bindata in the build. I will examine the failed job.

**Comment by idealhack** [comment]
I'm not sure how we review cli l10n, but...
@kubernetes/sig-cli-maintainers

File: translations/kubectl/ja_JP/LC_MESSAGES/k8s.po
Comment (k-toyoda-pi) [inline translations/kubectl/ja_JP/LC_MESSAGES/k8s.po]: I have checked japanese translation.
I think that "Display" means "表示する".

Comment (s-ito-ts) [inline translations/kubectl/ja_JP/LC_MESSAGES/k8s.po]: I think that this message is "リソースの説明を表示する".

**Comment by soltysh** [comment]
I don't have the necessary knowledge to verify correctness of these translation, so I'll defer to somehow who has. Once this passes the initial review I'll be happy to approve the PR.

File: translations/kubectl/ja_JP/LC_MESSAGES/k8s.po
Comment (tsasaki609) [reply translations/kubectl/ja_JP/LC_MESSAGES/k8s.po]: You're right.
I think that the translation of whole message is
"ポッドのリソース使用率(CPU/メモリ/ストレージ)を表示する。".

Comment (tsasaki609) [reply translations/kubectl/ja_JP/LC_MESSAGES/k8s.po]: In that case "(CPU/Memory/Storage)" will be omitted, but is it good?

Comment (s-ito-ts) [reply translations/kubectl/ja_JP/LC_MESSAGES/k8s.po]: I commented on the following message.

2834  msgid "Documentation of resources"

It seems that incorrect line is displayed on "Conversation" page.
Could you check it on "Files changed" page.

Comment (tsasaki609) [reply translations/kubectl/ja_JP/LC_MESSAGES/k8s.po]: The message I mentioned above is "Display Resource (CPU/Memory/Storage) usage of pods."
As @s-ito-ts said on the other review, it seems that incorrect line is displayed on "Conversation" page...

> "Display one or many resources"
> I think that "Display" means "表示する".

Your review is correct.
"記述する" is a mistranslation.
I will fix it later. 😃

**Comment by k-toyoda-pi** [comment]
Could you change PR title? "cubectl" to "kubectl"

**Comment by tsasaki609** [comment]
Hi @k-toyoda-pi ,
I changed PR title and commit log.

**Comment by tsasaki609** [comment]
Hi @soltysh ,
The PR pass the initial review by @k-toyoda-pi and @s-ito-ts who seems to be native Japanese.
Could you approve?

**Review by soltysh — APPROVED** [review]
/approve
Please squash your commits into one and I'll push that forward.

**Comment by tsasaki609** [comment]
Hi @soltysh ,
Thank you for approve.
I squashed my commits into one.

**Review by soltysh — APPROVED** [review]
/lgtm

**Comment by tsasaki609** [comment]
It seemed to have failed many times to retry CI tests, so I rebase the PR to latest master branch.

**Review by soltysh — APPROVED** [review]
/lgtm
/approve

**Pull request merged**
